frontend/src/lib/api.ts

```diff
@@ -1232,8 +1232,12 @@ const api = {
         ): Promise<NotebookType> {
             return await new ApiRequest().notebook(notebookId).update({ data })
         },
-        async list(): Promise<PaginatedResponse<NotebookType>> {
-            return await new ApiRequest().notebooks().get()
+        async list(sessionRecordingId?: string): Promise<PaginatedResponse<NotebookType>> {
+            const apiRequest = new ApiRequest().notebooks()
```

Comment:
Just thinking for the future, should we more loosely type this? I.e. allow you to just pass in the `contains` property?

Reply:
Other idea - we could type this really well so that you always give the proper type as an argument so its like `async list(contains: { type: NotebookNodeType, attrs: Record<string, string> })` and then we derive the correct query param structure from that